openapi/workflow_execution_service.openapi.yaml

```diff
@@ -542,6 +546,22 @@ components:
           type: string
         state:
           $ref: '#/components/schemas/State'
+    RunListStatus:
+      title: RunListStatus
+      required:
+        - run_id
+      type: object
+      properties:
+        run_id:
+          type: string
+        state:
+          $ref: '#/components/schemas/State'
+        start_time:
+          type: string
+          description: When the run started executing, in ISO 8601 format "%Y-%m-%dT%H:%M:%SZ"
+        end_time:
+          type: string
+          description: When the run stopped executing (completed, failed, or cancelled), in ISO 8601 format "%Y-%m-%dT%H:%M:%SZ"
       description: Small description of a workflow run, returned by server during listing
```

Comment:
The description was essentially moved from `RunStatus` (which, following your proposed changes doesn't have a description) to `RunListStatus`. I think this line should be available both for `RunStatus` and `RunListStatus`.